types/office-js-preview/index.d.ts

```diff
@@ -9573,6 +9573,124 @@ declare namespace Office {
              */
             Removed = "removed"
         }
+        /**
+         * Specifies the category color.
+         * 
+         * [Api set: Mailbox Preview]
+         * 
+         * @remarks
+         * <table>
+         *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Compose or Read</td></tr>
+         * </table>
+         * 
+         * @beta
+         */
+        enum CategoryColor {
+            /**
+             * Default color
+             */
+            None,
+            /**
+             * Red
+             */
+            Red,
+            /**
+             * Orange
+             */
+            Orange,
+            /**
+             * Yellow
+             */
+            Yellow,
+            /**
+             * Green
+             */
+            Green,
+            /**
+             * Teal
+             */
+            Teal,
+            /**
+             * Olive
+             */
+            Olive,
+            /**
+             * Blue
+             */
+            Blue,
+            /**
+             * Purple
+             */
+            Purple,
+            /**
+             * Peach/Brown
+             */
+            Peach, // TODO: Confirm if this or Brown
```

Comment:
Let's track the TODOs internally.

Reply:
I've requested review from Engineering to provide this clarification.

Reply:
Clarification provided so TODOs removed